packages/web-components/src/anchored-region/fixtures/anchored-region.html

```diff
@@ -0,0 +1,14 @@
+<fluent-design-system-provider use-defaults>
+  <h1>Tooltip</h1>
+
+  <h4>Default</h4>
+  <div style="height: 400px; width: 400px; background: lightgray;">
+    <fluent-tooltip anchor="anchor-default">
+      Helpful text is helpful
+    </fluent-tooltip>
+
+    <fluent-button id="anchor-default" style="margin: 200px;">
+      anchor
+    </fluent-button>
+  </div>
+</fluent-design-system-provider>
```

Comment:
Should this be in anchored region or should this be in tooltip?

Reply:
Looks like tooltip has its own. Is this meant to be anchored region fixture or is that the one below?

Reply:
Updated examples to be fluent anchor region